### DISCUSSION TITLE
Marine Forecast

### DISCUSSION THREAD

**Discussion by carlosga01** [weather-gov/api#217] [Q&A]
Hello, I have been scouring the API documentation for an endpoint to retrieve marine forecasts (wave height, wind direction, wind gusts, etc), but have not found any. Every forecast endpoint seems to not support marine requests. Are the marine forecasts not available on the public API? Or have I missed something?

**Comment by scadergit** [comment]
The API does not support marine forecast, but we're working with the OPC to provide that soon (probably late spring/summer).

**Reply by hbod8** [reply]
Are there any updates on the marine forecasts?  If not, where is the documentation for the legacy systems?

**Comment by carlosga01** [comment]
Thanks for the quick response! Sounds good
